tools/countUserSolutions.js

```diff
@@ -19,12 +19,15 @@ mongoose.connection.once('open', function() {
       }
     });
 
-    Object.entries(usersScores).forEach(async ([_id, score]) => {
```

Comment:
It could stay in functional style
```
await Promise.all(Object.entries(usersScores).map(([_id, score]) => { return user.findOneAndUpdate( ... ) })
```